src/styles/index.scss

```diff
@@ -1,3 +1,113 @@
+@import "./variables";
+
 body {
   margin: 0;
+  font-family: $main-font;
+  color: $main-color;
+  font-size: 12px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__photo {
+    margin: 32px 19px 0;
+
+    img {
+      width: 160px;
+      height: 134px;
+    }
+  }
+
+  &__element {
+    margin: 37px 16px 0;
+  }
+
+  &__title {
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    color: $secondary-color;
+    font-size: 10px;
+    line-height: 14px;
+    margin-top: 0;
+    margin-bottom: 16px;
+  }
+
+  &__rewievs {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__rewievsTitle {
+    font-size: 10px;
+    line-height: 14px;
+    vertical-align: top;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__word {
+    color: $secondary-color;
+    line-height: 18px;
+  }
+
+  &__amound {
+    color: $main-color;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__link {
+    display: block;
+    padding: 12px 0;
+    margin: 16px;
+    border-radius: 5px;
+    background: $blue-accent;
+    color: $white;
+    text-align: center;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    text-decoration: none;
+
+    &:hover {
+      border-radius: 5px;
```

Comment:
If you do not change the border radius when hovering over this element relative to its normal state, then why do you write it here?